extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -147,11 +147,19 @@ export async function createAzureFunction(connectionString: string, schema: stri
 				.withAdditionalProperties(propertyBag)
 				.withConnectionInfo(connectionInfo).send();
 		} catch (e) {
+			let errorType = utils.getErrorType(e);
 			propertyBag.quickPickStep = quickPickStep;
-			exitReason = 'error';
-			void vscode.window.showErrorMessage(utils.getErrorMessage(e));
 
-			TelemetryReporter.createErrorEvent(TelemetryViews.CreateAzureFunctionWithSqlBinding, TelemetryActions.exitCreateAzureFunctionQuickpick, undefined, utils.getErrorType(e))
+			if (errorType === 'TimeoutError') {
```

Comment:
I'm not seeing a return here - shouldn't we be doing that since we're in an error state?